backend/src/main/java/com/github/mariakohnen/packbag/backend/service/PackingListService.java

```diff
@@ -90,5 +89,16 @@ public PackingList deleteItemFromPackingList(String id, String itemId) {
             return packingListRepository.save(updatedPackingList);
         } else throw new NoSuchElementException("There is no item with the id: " + itemId);
     }
+
+    public PackingList updatePackingItem(String id, String itemId, CreatePackingItemDto createPackingItemDto) {
+        if (createPackingItemDto.getName() == null) {
+            throw new IllegalArgumentException("Name of the given item was null.");
```

Comment:
Is it necessary to give some hint in the string that the item should be updated? Maybe not, because the console shows the method the Exception was thrown. What is your opinion?

Reply:
good idea. I will make the message a little more detailed